client/reader/user-profile/components/user-profile-header/test/index.tsx

```diff
@@ -0,0 +1,213 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import UserProfileHeader from '../index';
+
+// Mock external icon
+jest.mock( '@wordpress/icons', () => ( {
+	external: 'mock-external-icon',
+	Icon: ( { icon } ) => <span data-testid="icon">{ icon }</span>,
+} ) );
+
+// Mock ReaderAvatar component
+jest.mock( 'calypso/blocks/reader-avatar', () => ( {
+	__esModule: true,
+	default: ( { author, iconSize } ) => (
+		<div data-testid="reader-avatar" data-author-id={ author.ID } data-icon-size={ iconSize }>
+			{ author.avatar_URL && (
+				<img src={ author.avatar_URL } alt="avatar" data-testid="avatar-img" />
+			) }
+		</div>
+	),
+} ) );
```

Comment:
We can simplify all of these mock statements as following i.. avoiding __esModule and default attributes.

```suggestion
jest.mock( 'calypso/blocks/reader-avatar', () => ( { author, iconSize } ) => (
	<div data-testid="reader-avatar" data-author-id={ author.ID } data-icon-size={ iconSize }>
		{ author.avatar_URL && <img src={ author.avatar_URL } alt="avatar" data-testid="avatar-img" /> }
	</div>
) );
```

Reply:
Simplified in 0122f80cd8ebd4c1ef4f6ef4ca9f9db55c8ae813